cognite/client/_api/transformation_notifications.py

```diff
@@ -0,0 +1,117 @@
+    ) -> TransformationNotificationList:
+        """`List notification subscriptions. <https://docs.cognite.com/api/playground/#operation/listTransformationNotifications>`_
+
+        Args:
+            transformation_id (Optional[int]): List only notifications for the specified transformation. The transformation is identified by internal numeric ID.
+            transformation_external_id (str): List only notifications for the specified transformation. The transformation is identified by externalId.
+            destination (str): Filter by notification destination.
+            limit (int): Limits the number of results to be returned. To retrieve all results use limit=-1, default limit is 25.
+
+        Returns:
+            TransformationNotificationList: List of transformation notifications
+
+        Example:
+
+            List all notifications::
+
+                >>> from cognite.experimental import CogniteClient
+                >>> c = CogniteClient()
+                >>> notifications_list = c.transformations.notifications.list()
+
+            List all notifications by transformation id::
+
+                >>> from cognite.experimental import CogniteClient
+                >>> c = CogniteClient()
+                >>> notifications_list = c.transformations.notifications.list(transformation_id = 1)
+
+            List all notifications by transformation external id::
+
+                >>> from cognite.experimental import CogniteClient
+                >>> c = CogniteClient()
+                >>> notifications_list = c.transformations.notifications.list(transformation_external_id = "myExternalId")
+
+            List all notifications by destination::
+
+                >>> from cognite.experimental import CogniteClient
+                >>> c = CogniteClient()
+                >>> notifications_list = c.transformations.notifications.list(destination = "my@email.com")
+        """
+        filter = TransformationNotificationFilter(
+            transformation_id=transformation_id,
+            transformation_external_id=transformation_external_id,
+            destination=destination,
+        ).dump(camel_case=True)
+
+        return self._list(method="GET", limit=limit, filter=filter)
+
+    def delete(self, id: Union[int, List[int]] = None) -> None:
+        """`Deletes the specified notification subscriptions on the transformation. Does nothing when the subscriptions already don't exist <https://doc.cognitedata.com/api/playground/#operation/deleteTransformationNotifications>`_
+
+        Args:
+            id (Union[int, List[int]): Id or list of ids
+
+        Returns:
+            None
+
+        Examples:
+
+            Delete schedules by id or external id::
+
+                >>> from cognite.experimental import CogniteClient
+                >>> c = CogniteClient()
+                >>> c.transformations.notifications.delete(id=[1,2,3])
+        """
+        self._delete_multiple(ids=id, wrap_ids=True)
```

Comment:
might as well add 
`utils._auxiliary.assert_type(ids, "id", [List], allow_none=True)`

Reply:
type assertion is done inside `base._delete_multiple()`, id can be a single integer or a list of integers, and it will handle the cases inside the base logic.